SPECS/dtc/dtc.spec

```diff
@@ -35,19 +39,27 @@ This package provides development files for libfdt
 
 %prep
 %autosetup -p1
-sed -i 's/python2/python3/' pylibfdt/setup.py
-sed -i 's/SUBLEVEL = 0/SUBLEVEL = 1/' Makefile
+# to prevent setuptools from installing an .egg, we need to pass --root to setup.py install
+# since $(PREFIX) already contains %%{buildroot}, we set root to /
+# .eggs are going to be deprecated, see https://github.com/pypa/pip/issues/11501
+sed -i 's@--prefix=$(PREFIX)@--prefix=$(PREFIX) --root=/@' pylibfdt/Makefile.pylibfdt
 
 %build
+export SETUPTOOLS_SCM_PRETEND_VERSION=%{version}
```

Comment:
Could you add a comment explaining why this `export` is needed, so it's easier to understand in the future?

Reply:
Done! Also while this is the recommended way for this release (according to dtc). It looks like it will change in future releases so I also mentioned that :)

Reply:
Here is a link to the referenced commit: https://git.kernel.org/pub/scm/utils/dtc/dtc.git/commit/setup.py?id=cd3e2304f4a9c4f725dcc548082ccc6c89d078a4